modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm

```diff
@@ -184,8 +185,11 @@
 
 /obj/item/organ/genital/vagina/get_description_string(datum/sprite_accessory/genital/gas)
```

Comment:
I just hope that the sprite accessory is actually being passed as an argument or I'm gonna be big sad